bindings/netstandard/ElectionGuard/ElectionGuard.Encryption/DisposableBase.cs

```diff
@@ -6,7 +6,7 @@ namespace ElectionGuard
     /// </summary>
     public class DisposableBase : IDisposable
     {
-        private bool _disposedValue;
+        public bool IsDisposed { get; private set; }
```

Comment:
it could probably be internal now that i added `IsAddressable` however its useful to be able to check the state of these objects that wrap native code